### PR TITLE
[alamin step-5] added planning docs

### PR DESCRIPTION
**Pull request by rr-mdalauddin-alamin**
**Step 5: Think about the image of the application you want to create**

- Before proceeding with the design, think about what kind of app it will be (with your mentor). Screen design by paper prototyping is recommended.
- Read the system requirements and think about the required data structures
    - What kind of model (table) do you need?
    - What information do you want in the table
- Once you think of the data structure, let's draw it on the model diagram by hand.
    - Take a picture when you're done and put it in your repository
    - README.mdLet's describe the table schema in (model name, column name, data type)

※  There is no need to create a correct model diagram at this time. Let's make it as an assumption at the moment (it is an image that will be repaired if you think it is a mistake in the future steps)

### CODE REVIEW

File: myapp/docs/ERD.png
Comment (stakahashi-d) [inline myapp/docs/ERD.png:1]: The bottom right table's name are a mistake, maybe `tasks_labels`?

**Review by stakahashi-d — COMMENTED** [review]
Most of part is OK, but I left a one comment !
Please fix that and I can approve!

And also copy and paste the step content to your PR description, so that reviewer can easily know what you are doing!

File: myapp/docs/ERD.png
Comment (rr-mdalauddin-alamin) [reply myapp/docs/ERD.png:1]: ah! yep!

Comment (rr-mdalauddin-alamin) [reply myapp/docs/ERD.png:1]: fixed

**Review by stakahashi-d — APPROVED** [review]
LGTM

**Review by stakahashi-d — APPROVED** [review]

**Pull request merged**